packages/mdc-select/mdc-select.scss

```diff
@@ -80,7 +80,7 @@
   }
 
   &__menu {
-    position: fixed;
+    position: absolute;
```

Comment:
Why are you changing the position fixed to position absolute? It's not necessary as the body will not be scrollable :smiley:

Reply:
You're right :smile: I'm pushing the commit that reverts the position value